TwitchClient/GamesListViewController.swift

```diff
@@ -1,108 +0,0 @@
-//
-//  GamesListViewController.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/7/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-import SVProgressHUD
-
-class GamesListViewController: UIViewController, WebserviceManagerDelegate, UITableViewDelegate, UITableViewDataSource {
-
-    
-    @IBOutlet weak var gamesTableView: UITableView!
-    
-    
-    var games = [Game]()
-   
-    var selectedGame = Game()
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
-        
-        self.gamesTableView.isHidden = true
-        SVProgressHUD.show()
-        
-        self.getTopGames()
-        
-    }
-   
-    func getTopGames()
-    {
-        // initialize webservice manager and call the web service request
-        let manager = WebServiceManager()
-        manager.delegate = self
-        manager.getTopGames()
-    }
-    //MARK:- Webservice Manager Delegate
-    func didFinishRetrievingData(sender: WebServiceManager, GamesList: [Game]?, StreamList: [Stream]?, error: Bool) {
-        
-        if error == false{
```

Comment:
`guard if error else { 
showAlert()
return
 }`
Then have the normal "happy path" outside the bracket.

Topics: Golden/happy path, clean code.